app/src/main/java/org/gnucash/android/model/ScheduledAction.java

```diff
@@ -322,6 +327,11 @@ public int getExecutionCount(){
 
     /**
      * Sets the number of times this scheduled action has been executed
+     *
+     * <p><b>Important:</b> when updating this field include not only the real
```

Comment:
Yes, I agree. It was a last minute change and I really don't like it either. Specially taking into account that it only applies to scheduled exports.